src/ape/api/providers.py

```diff
@@ -1075,7 +1089,12 @@ def start(self, timeout: int = 20):
             with RPCTimeoutError(self, seconds=timeout) as _timeout:
                 while True:
                     if self.is_connected:
-                        break
+                        chain_id = self.chain_id  # Ensure this is how chain_id is accessed
+                    if chain_id == 1337:
```

Comment:
The chain ID is configurable and does not mean it is an Ethereum tester node. Any node can use this port!